src/Forms/Forms.test.js

```diff
@@ -1,7 +1,17 @@
+import FormFieldset from './FormFieldset';
```

Comment:
Ideally, these tests would be broken into separate `*.test.js` files -- one per component.  That said, a number of tests will need to be written or rewritten to test individual components.  I created #429 to track this work.